index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./stylesheet/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1 class="text-center main-logo">My Library</h1>
+    <button id="addBookBtn" class="center-it">Add Book</button>
+    <div id="form" class="hide">
+      <label for="title">Title</label>
+
+      <input type="text" name="title" id="title" />
+      <br />
+      <label for="author">Author</label>
+
+      <input type="text" name="author" id="author" />
+      <br />
+      <label for="pages">Pages</label>
+
+      <input type="number" name="pages" id="pages" />
+      <br />
+      <label for="read">Read</label>
+
+      <input type="checkbox" name="read" id="read" />
+      <br />
+      <button type="submit" id="create">Create Book</button>

```

Comment:
Opps, i don't think you were having this issue before..... 🤔 
Kindly clear the form once the user creates a book.
(I need to clear the form manually to add a new book.)
![image](https://user-images.githubusercontent.com/31897434/87504556-be4a7e80-c65e-11ea-99b9-50312c8f43b0.png)